### ISSUE TITLE
Add default Natvis for GCC STL

### ISSUE THREAD

**Issue by yh-sb** [microsoft/vscode-cpptools#10993]
### Feature Request

As you know, `cppvsdbg` has default Natvis for STL unlike `cppdbg` GCC. I would like to suggest adding a default visualizer for GCC STL in case when `"visualizerFile"` isn't set in launch.json.

**Here is my Natvis file for GCC STL (I tested it with GCC 12 and 13):** https://gist.github.com/yhsb2k/5c56887cd6d8f06f54846fa11c613748

**Comment by WardenGnaw** [comment]
Ideally, GDB's pretty printer would handle visualizing standard library objects. Are there objects that are not being properly handled by GDB's pretty printer?

**Comment by yhsb2k** [comment]
GDB's pretty printer is really printing its STL objects correctly.
Source code:
```c++
std::vector<int> vect = {2, 1, 3, 20};
```
GDB terminal:
```
print vect
$1 = std::vector of length 4, capacity 4 = {2, 1, 3, 20}
```

But vscode doesn't use this.

**Comment by david-fong** [comment]
@yh-sb have you added `-enable-pretty-printing` to your launch config's `setupCommands`?

**Comment by yh-sb** [comment]
@david-fong, yes, it doesn't help. Here is what I get with `-enable-pretty-printing` on `std::vector<int>`:

<img width="2286" height="672" alt="Image" src="https://github.com/user-attachments/assets/214a7caf-d6c8-4638-9e74-2427b57c3319" />

[debug_console.log](https://github.com/user-attachments/files/22990072/debug_console.log)
